tests/test_measure.py

```diff
@@ -3,21 +3,42 @@
 
 from btk.catalog import CatsimCatalog
 from btk.draw_blends import CatsimGenerator
-from btk.measure import MeasureGenerator, basic_measure, sep_measure
+from btk.measure import (
+    MeasureGenerator,
+    basic_measure,
+    sep_multiband_measure,
+    sep_singleband_measure,
+)
 from btk.sampling_functions import DefaultSampling
 from btk.survey import get_surveys
 
 TEST_SEED = 0
 
 
 def get_meas_results(meas_function, cpus=1, measure_kwargs=None):
-    """Returns draw generator with group sampling function"""
-
+    """Runs a measurement function on a set of data -- returns targets and predicitons"""
     catalog_name = data_dir / "sample_input_catalog.fits"
     stamp_size = 24
     survey = get_surveys("LSST")
-    shifts = [[-0.3, 1.2]]
-    indexes = [[1]]
+    shifts = [
```

Comment:
I assume you added all these shifts to test it in  a lot more objects? :)

Reply:
How did you pick them btw?

Reply:
Yeah... I just found these shifts in `test_metrics.py` and changed it to have one galaxy in a blend, so it would be easier to match sep detection to the ground truth.

Reply:
great that makes sense to me, thank you!